homeassistant/config_entries.py

```diff
@@ -780,7 +788,7 @@ async def _async_init_reauth(
             if any(self.async_get_active_flows(hass, {SOURCE_REAUTH})):
                 # Reauth flow already in progress for this entry
                 return
-            await hass.config_entries.flow.async_init(
+            result = await hass.config_entries.flow.async_init(
```

Comment:
This won't return until the flow is done. That means we'll create a repair issue after the reauth flow is completed.